source/code-snippets/crud/bulk.js

```diff
@@ -1,117 +1,252 @@
+    });
 
     try {
         await client.connect();
-        
+
         const database = client.db('sample_mflix');
         const movies = database.collection('movies');
 
-        // Clean up collection
-        await movies.deleteMany({});
-
-        // begin-sample-data
-        // const movies = database.collection('movies');
-        
-        const docs = [
-        { title: "Inception", year: 2010, rated: "PG-13", released: "2010-07-16" },
-        { title: "Interstellar", year: 2014, rated: "PG-13", released: "2014-11-07" },
-        { title: "The Dark Knight", year: 2008, rated: "PG-13", released: "2008-07-18" },
-        { title: "Tenet", year: 2020, rated: "PG-13", released: "2020-09-03"}
-        ];
-        // end-sample-data
-
-        // begin-insert
-        const bulkOps = [
-        { insertOne: { document: { title: "Inception", year: 2010, rated: "PG-13", released: "2010-07-16" } } },
-        { insertOne: { document: { title: "Interstellar", year: 2014, rated: "PG-13", released: "2014-11-07" } } },
-        { insertOne: { document: { title: "The Dark Knight", year: 2008, rated: "PG-13", released: "2008-07-18" } } },
-        { insertOne: { document: { title: "Tenet", year: 2020, rated: "PG-13", released: "2020-09-03" } } }
-          ];
-
-        await movies.bulkWrite(bulkOps);
-        // end-insert
-
-        await movies.insertMany(docs);
-
-        // Inserting additional movies
-        const additionalMovies = [
-        { title: "Dunkirk", year: 2017, rated: "PG-13", released: "2017-07-21" },
-        { title: "Memento", year: 2000, rated: "R", released: "2000-09-05" }
-        ];
-        await movies.insertMany(additionalMovies);
-
-
-        // begin-replace
-        const replaceOperations = [
-            {
-                replaceOne: {
-                    filter: { title: "The Dark Knight" },
-                    replacement: { title: "The Dark Knight Rises", year: 2012, rating: "PG-13" },
-                    upsert: false
+        // begin-insert-coll
+        const insertModels = [{
+            insertOne: {
+                document: {
+                    title: "The Favourite",
+                    year: 2018,
+                    rated: "R",
+                    released: "2018-12-21"
                 }
-            },
-            {
-                replaceOne: {
-                    filter: { title: "Inception" },
-                    replacement: { title: "Inception Reloaded", year: 2010, rating: "PG-13" },
-                    upsert: false
+            }
+        }, {
+            insertOne: {
+                document: {
+                    title: "I, Tonya",
+                    year: 2017,
+                    rated: "R",
+                    released: "2017-12-08"
                 }
             }
-        ];
+        }];
+
+        const insert_result = await movies.bulkWrite(insertModels);
+        console.log(`Inserted documents: ${insert_result.insertedCount}`);
```

Comment:
```suggestion
        const insertResult = await movies.bulkWrite(insertModels);
        console.log(`Inserted documents: ${insertResult.insertedCount}`);
```

JS conventions don't use snake case.